docs/topics/proc_starting-an-uberjar-in-debugging-mode.adoc

```diff
@@ -19,15 +19,14 @@ If you chose to package your application as a {runtime} uberjar, debug it by exe
 . Execute the uberjar with the following parameters.
 Ensure that all the parameters are specified before the name of the uberjar on the line.
 +
---
 [source,bash,options="nowrap"]
 ----
-$ java -agentlib:jdwp=transport=dt_socket,server=y,suspend=y,address=$PORT_NUMBER -jar $UBERJAR_FILENAME
+$ java -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=$PORT_NUMBER -jar $UBERJAR_FILENAME
 ----
-
-Here, `$PORT_NUMBER` is an unused port number of your choice.
-Remember this number for the remote debugger configuration.
---
++
+`$PORT_NUMBER` is an unused port number of your choice. Remember this number for the remote debugger configuration.
++
+If you want the application to pause immediately, change `suspend` to `y`.
```

Comment:
"Immediately" is very vague here. I'd rephrase to something like "If you want the JVM to pause and wait for remote debugger connection before it starts the application".

Reply:
+1